src/test/java/AirportTest.java

```diff
@@ -35,98 +35,75 @@
         for (int i = 0; i < planesSortedByMaxLoadCapacity.size() - 1; i++) {
             Plane currentPlane = planesSortedByMaxLoadCapacity.get(i);
             Plane nextPlane = planesSortedByMaxLoadCapacity.get(i + 1);
             if (currentPlane.getMaxLoadCapacity() > nextPlane.getMaxLoadCapacity()) {
-                nextPlaneMaxLoadCapacityIsHigherThanCurrent = false;
-                break;
+               return false;
             }
         }
-        return nextPlaneMaxLoadCapacityIsHigherThanCurrent;
+        return true;
     }
 
 
-    private boolean testisBomberMilitaryPlane(List<MilitaryPlane> bomberMilitaryPlanes, boolean isbomberMilitaryPlane) {
+    private boolean isBomberMilitaryPlane(List<MilitaryPlane> bomberMilitaryPlanes) {
         for (MilitaryPlane militaryPlane : bomberMilitaryPlanes) {
-            if ((militaryPlane.getType() == MilitaryType.BOMBER)) {
-                isbomberMilitaryPlane = true;
+            if ((militaryPlane.getType() != MilitaryType.BOMBER)) {
+               return false;
             }
         }
-        return isbomberMilitaryPlane;
-    }
-
-    @Test
-    public void testGetTransportMilitaryPlanes() throws NonMilitaryPlainException {
-        Airport airport = new Airport(planes);
-        List<MilitaryPlane> transportMilitaryPlanes = airport.getMilitaryPlanesByType(MilitaryType.TRANSPORT);
-        boolean militaryTypeIsTransport = false;
-        Assert.assertEquals(testGetTransportMilitaryPlanes(transportMilitaryPlanes, militaryTypeIsTransport), true);
+        return true;
     }
 
 
     @Test
-    public void testGetPassengerPlaneWithMaxCapacity() throws NonMilitaryPlainException {
+    public void testGetPassengerPlaneWithMaxCapacity()  {
         Airport airport = new Airport(planes);
         PassengerPlane expectedPlaneWithMaxPassengersCapacity = airport.getPassengerPlaneWithMaxPassengersCapacity();
-        Assert.assertTrue(expectedPlaneWithMaxPassengersCapacity.equals(planeWithMaxPassengerCapacity));
+        Assert.assertEquals(expectedPlaneWithMaxPassengersCapacity, planeWithMaxPassengerCapacity);
     }
 
+
     @Test
-    public void testNextPlaneMaxLoadCapacityIsHigherThanCurrent() throws NonMilitaryPlainException {
+    public void testHasAtLeastOneBomberInMilitaryPlanes(){
         Airport airport = new Airport(planes);
-        airport.sortByMaxLoadCapacity(planes);
-        List<? extends Plane> planesSortedByMaxLoadCapacity = airport.getPlanes();
-        boolean nextPlaneMaxLoadCapacityIsHigherThanCurrent = true;
-
-        {
-            if (!(testComparasionNextPlaneWithCurrent(planesSortedByMaxLoadCapacity, nextPlaneMaxLoadCapacityIsHigherThanCurrent)))
-                Assert.fail("Test failed!");
+        List<MilitaryPlane> bomberMilitaryPlanes = airport.getMilitaryPlanesByType(MilitaryType.BOMBER);
+        Assert.assertTrue(isBomberMilitaryPlane(bomberMilitaryPlanes));
```

Comment:
test doesn't correspond to what is doing
you're sending a list full of bombers. it should fail if at least 1 plane is not bomber in list.
test should be named smth like "testMilitaryBomberPlanesListContainsOnlyBombers"